miniapps/meshing/makefile

```diff
@@ -108,14 +108,14 @@ clean: clean-build clean-exec
 
 clean-build:
 	rm -f *.o *~ mobius-strip klein-bottle toroid twist
-	rm -f mesh-explorer shaper extruder trimmer
+	rm -f mesh-explorer shaper extruder trimmer reflector
```

Comment:
Should probably add `reflector` to `SEQ_MINIAPPS` above.

Reply:
Initially, I did add `reflector` to `SEQ_MINIAPPS`, but then I reverted that change in commit eadf460df55266d9b6f388ae8cab768e38416e8e, because it caused a CI failure. I added it back, and the CI is failing again. I could use some advice on how to fix this issue.

Reply:
I believe the issue is that `reflector` does not fit the expected pattern in that it produces a file named `reflected.mesh` rather than `reflector.mesh`. The script that produces the test runs assumes this pattern and since `reflector.mesh` doesn't exist it assumes the run failed. The `toroid` and `twist`miniapps also breaks this pattern.

Unfortunately, I don't think the script (located in `config/test.mk`) contains documentation explaining its assumptions but I think the following will work. Add a special `make` target for the `reflector` test which provides the name of the expected output file:
```
reflector-test-seq: %
	@$(call mfem-test-file,$<,, Meshing miniapp,reflected.mesh)
```

Reply:
There may still be other issues but this should help.

Reply:
Thanks, a slight variation of this worked.